src/main/kotlin/org/vaccineimpact/api/models/CoverageIngestionRow.kt

```diff
@@ -0,0 +1,13 @@
+package org.vaccineimpact.api.models
+
+data class CoverageIngestionRow(
```

Comment:
I guess this is derived from the spec we sent to gavi already, but just noticed we also have gender in the db, but not here. Is that because all gavi coverage is for both genders? There's also 'gavi support level' at the coverage set level, will that also be fixed for all gavi provided coverage, or can be derived from the support levels found in the rows?

Reply:
Oh, good question re gender, I'll check that with the science team. Re the coverage sets, they will be derived from the rows - each combination of support level, activity type and vaccine determines a coverage set.

Reply:
Added gender :+1: